packages/airnode-validator/README.md

```diff
@@ -1,16 +1,13 @@
 # `@api3/airnode-validator`
 
-The validator is used by the Docker Images to validate the configuration files you provide when deploying an Airnode.
-You can also use the validator to check the configuration files for correct formatting and other issues while creating
-them.
+> API and CLI for validating Airnode configuration files
 
-## Usage
+## User documentation
 
-<!-- TODO: Link to validator docs when available -->
+<!-- TODO: Add correct link to validator -->
```

Comment:
I don't understand. What is this TODO for?

Reply:
For the text below:

```
You can find documentation on how to use the deployer in the [validator docs](https://docs.api3.org/airnode/v0.7/).
```

but I now realized it says "deployer" instead of validator.

Reply:
But there is already a link. What am I missing? :smile:

Reply:
It doesn't point to validator, but to the root documentation.